packages/client/hmi-client/src/components/workflow/ops/funman/funman-operation.ts

```diff
@@ -51,8 +52,7 @@ export interface FunmanOperationState extends BaseState {
 	compartmentalConstraint: CompartmentalConstraint;
 	constraintGroups: ConstraintGroup[];
 	requestParameters: RequestParameter[];
-	// selected state in ouptut
-	trajectoryState?: string;
+	chartSettings: ChartSetting[] | null;
```

Comment:
Maybe check with @jryu01 whether the chart-settings should be null or a blank array.

Reply:
It starts as `null` in the operations where he used it